From 73a776f76043d122438d716d0958489fc67bab8e Mon Sep 17 00:00:00 2001
From: Bernd Waibel <waebbl@gmail.com>
Date: Sun, 1 Sep 2019 22:02:42 +0200
Subject: [PATCH] doc/html.doc/Makefile.am: install into $htmldir

Patches the install-data-local target to use $htmldir instead of $docdir

Signed-off-by: Bernd Waibel <waebbl@gmail.com>
---
 doc/html.dox/Makefile.am | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/doc/html.dox/Makefile.am b/doc/html.dox/Makefile.am
index 9dc5b08..3946777 100644
--- a/doc/html.dox/Makefile.am
+++ b/doc/html.dox/Makefile.am
@@ -36,25 +36,25 @@ html-local: $(builddir)/../dox/Doxyfile.cfg
 	diff $(builddir)/htmllistfile2.am.tmp $(srcdir)/htmllistfile2.am || @CP@ $(builddir)/htmllistfile2.am.tmp $(srcdir)/htmllistfile2.am
 
 install-data-local: $(htmllistfile1) $(htmllistfile2) $(htmllistfile3)
-	test -z "$(DESTDIR)$(docdir)" || $(MKDIR_P) "$(DESTDIR)$(docdir)"
+	test -z "$(DESTDIR)$(htmldir)" || $(MKDIR_P) "$(DESTDIR)$(htmldir)"
 	for i in $(htmllistfile1) ; do \
 	  $(INSTALL_DATA) \
 	     `test -f $$i || echo $(srcdir)/`$$i \
-	     $(DESTDIR)$(docdir) ; \
+	     $(DESTDIR)$(htmldir) ; \
 	done 
 	for i in $(htmllistfile2) ; do \
 	  $(INSTALL_DATA) \
 	     `test -f $$i || echo $(srcdir)/`$$i \
-	     $(DESTDIR)$(docdir) ; \
+	     $(DESTDIR)$(htmldir) ; \
 	done
 	for i in $(htmllistfile3) ; do \
 	  $(INSTALL_DATA) \
 	     `test -f $$i || echo $(srcdir)/`$$i \
-	     $(DESTDIR)$(docdir) ; \
+	     $(DESTDIR)$(htmldir) ; \
 	done
 
 uninstall-local: $(htmllistfile1) $(htmllistfile2) $(htmllistfile3)
-	rm -rf $(DESTDIR)$(docdir); 
+	rm -rf $(DESTDIR)$(htmldir); 
 
 .PHONY: htmlclean
 
-- 
2.22.0

